reflex/components/radix/primitives/accordion.py

```diff
@@ -409,7 +336,7 @@ class AccordionRoot(AccordionComponent):
     variant: Var[LiteralAccordionRootVariant] = "classic"  # type: ignore
 
     # The color scheme of the accordion.
-    color_scheme: Var[LiteralAccordionRootColorScheme] = "primary"  # type: ignore
+    color_scheme: Var[LiteralAccentColor] = "green"  # type: ignore
```

Comment:
```suggestion
    color_scheme: Var[LiteralAccentColor]  # type: ignore
```

i dont think there should be a default here. if `color_scheme` is unspecified, then it should use the theme color right?